common/functions.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * The purpose of this function is to determine which hour is nearest to the hour in which the funciton is called.
+ */
+function getHourDivisibleByThree (hours) {
+  const remainder = hours % 3;
+  let currentHour = hours;
+
+  const CurrentHourIsNotDivisibleByThree = remainder !== 0;
+
+  if (CurrentHourIsNotDivisibleByThree) {
+    const isCurrentHourNearestToNextHour = remainder > 1;
+    if (isCurrentHourNearestToNextHour) currentHour += 1;
+    else currentHour -= 1;
+  }
+  currentHour = currentHour === 24 ? 0 : currentHour
```

Comment:
this code should be at line 7 and this should go like this 
`if(current === 24) return 0`

Reply:
This can't be on line 7, as `currentHour` will become 24 after the conditional statement is executed.